src/udev/udevd.c

```diff
@@ -289,18 +289,14 @@ int run_udevd(int argc, char *argv[]) {
         int fd_ctrl = -EBADF, fd_uevent = -EBADF;
         int r;
 
-        log_set_target(LOG_TARGET_AUTO);
-        log_open();
+        log_setup();
 
         manager = manager_new();
         if (!manager)
                 return log_oom();
 
         manager_parse_udev_config(manager);
 
-        log_parse_environment();
-        log_open(); /* Done again to update after reading configuration. */
```

Comment:
Why remove this? The log target might have changed after parsing udev.conf

Reply:
@YHNdnzj We don't seem to read a log target from udev.conf? See `manager_parse_udev_config()`

Reply:
Hmm, right. Sorry for the noise.